docs/reference/server-configuration.md

```diff
@@ -10,10 +10,16 @@ keywords: ksqldb, configure, server, setup, install
 
 These configuration parameters control the general behavior of ksqlDB server.
 Many parameters can only be set once for the entire server, and must be
-specified using the `ksql-server.properties` file. Some parameters, however,
-can be set on a per-persistent query basis using `SET`. This is indicated in each parameter
-section below.
-Retrieve the current list of configuration settings by using the [SHOW PROPERTIES](/developer-guide/ksqldb-reference/show-properties/) command.
+specified using the `ksql-server.properties` file.
+
+You can assign the value of some parameters on a per-persistent query basis
```

Comment:
@rodesai does `SET` work on push queries or can you not configure them at all?

Reply:
Good point to generalize to all queries. Also with shared runtime, we changed the definition of "per query", so maybe we need to rephrase a little.
@JimGalasyn , what about something like to make this clear:
...
specified using the `ksql-server.properties` file (for on-prem / standalone). In this case, configuration are applied when the cluster starts.
A subset of these configuration parameters can be applied on a running cluster, either for individual queries (using the SET command or the UI), or for the entire cluster (using ALTER SYSTEM command or the UI). If this is the case, this will be mentioned the list below. Note that only this subset can be edited in Confluent Cloud at this time.

Reply:
> Note that only this subset can be edited in Confluent Cloud at this time.

@JSeb225 should this say "Note that **this subset can only** be edited in Confluent Cloud at this time"?